src/sage/coding/channel.py

```diff
@@ -6,9 +6,10 @@
 input space (the message) and transforms it into an element of the output space
 (the transmitted message).
 
-In Sage, Channels simulate error-prone transmission over communication
-channels, and we borrow the nomenclature from communication theory, such as
-"transmission" and "positions" as the elements of transmitted vectors.
+In SageMath, a :class:`Channel` object simulates error-prone transmission over
+communication channels. We borrow the nomenclature from communication theory,
+such as "transmission" and "positions" as the elements of transmitted vectors.
+
```

Comment:
"SageMath" is the name of Sage for outside people to avoid confusion with other software also named "Sage".

Within us, we just say "Sage". I think we should use "Sage" universally in the Sage documentation.

I like to think of it as "Sage Math" where "Sage" is the first name and "Math" the last name :-)